index.html

```diff
@@ -1,48 +1,122 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title> Golpes Básicos </title>
+        <title> Bases del Boxeo </title>
         <meta charset="UTF-8">
-        <meta name="description" content="Aprende los golpes basicos del deporte">
+        <meta name="description" content="Aprende las bases del ">
+        <link rel="stylesheet" href="style/style.css">
     </head> 
-    <body style="background-color: darkslategrey;">
+    <body>
        <header>
-           <h1 style="color: maroon;"> El Boxeo</h1>
+           <h1 class="titles"> El Boxeo</h1>
            <hr>
+           <h1 class="subtitulo">Golpes</h1>
            <h2><p>El boxeo es un deporte de contacto en donde dos contrincantes se enfretan solamente usando sus puños</p></h2>
            <h2><p>Al solo utilizarse los puños la cantidad de golpes posibles son limitados. Es por eso que es facil listarlos</p> </h2>
-           <h2><p>A continuacion se encuentran los golpes mas importantes y remarcables del boxeo con su respectiva demostración y descripción</p></h2>           
+           <h2><p>A continuacion se encuentran los golpes mas importantes y remarcables del boxeo con su respectiva demostración y descripción</p></h2>
+           <hr>
         </header>
-        <hr>
+        
         <main>
             
             <dl>
-                <h1><dt>Jab</dt></h1>
-                <h2><dd>Un puñetazo veloz y directo, lanzado con la mano delantera desde la posición de guardia. El jab es acompañado de una pequeña rotación del torso y la cadera, en el sentido de las agujas del reloj, mientras que el puño rota 90 grados, adquiriendo una línea de golpe horizontal en los nudillos en el momento del impacto. A medida que el puñetazo alcanza su mayor magnitud, el hombro en posición delantera se antepone al mentón protegiéndolo. La mano que se encuentra en posición trasera permanece junto al rostro para proteger la mandíbula</dd></h2>
-                <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1c/Jab3.jpg/800px-Jab3.jpg"width="200">
+                <h2 class="nombres"><dt>Jab</dt></h2>
+                <div class="div2">
+                    <dd class="descrip">Un puñetazo veloz y directo, lanzado con la mano delantera desde la posición de guardia. El jab es acompañado de una pequeña rotación del torso y la cadera, en el sentido de las agujas del reloj, mientras que el puño rota 90 grados, adquiriendo una línea de golpe horizontal en los nudillos en el momento del impacto. A medida que el puñetazo alcanza su mayor magnitud, el hombro en posición delantera se antepone al mentón protegiéndolo. La mano que se encuentra en posición trasera permanece junto al rostro para proteger la mandíbula</dd>
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1c/Jab3.jpg/800px-Jab3.jpg"; class="imagenes">
```

Comment:
Replace url images with local ones, also remove the `;` since it's not necessary

Reply:
And do this to every image in this file